client/src/renderer/components/ShortcutListener.js

```diff
@@ -0,0 +1,51 @@
+/** @format */
+
+import { Component, h } from "preact";
+import { connect } from "react/redux";
+import matchEvent from "../commands/shortcuts.js";
+
+/**
+ * Non-rendering component, responsible for handling keyboard shortcuts
+ * @extends Component
+ */
+
+class ShortcutListener extends Component {
+  constructor(props) {
+    super(props);
+    this.handleKeyPress = this.handleKeyPress.bind(this);
+  }
+
+  /**
+   * On mount, add keydown event listener.
+   * On event, handleKeyPress().
+   * On unmount, remove listener.
+   */
+  componentDidMount() {
+    window.addEventListener("keydown", this.handleKeyPress, true);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener("keydown");
```

Comment:
Be careful to only remove the event listener you created. This will remove all event listeners on "keydown"

```suggestion
    window.removeEventListener("keydown", this.handleKeyPress);
```